### ISSUE TITLE
Feature request: crossing thumbs should move both thumbs in parallel

### ISSUE THREAD

**Issue by avlonder** [Sharcoux/slider#14]
Apart from the crossing allowed prop, is it also possible to add a prop that defines the minimum distance between the two thumbs and if you drag the left thumb beyond the right one, both of them are moving in parallel?

**Comment by Sharcoux** [comment]
Would you be OK to try a PR achieving this? I won't be able to work on this before 2 weeks I think, but it should not be so hard.

**Comment by Sharcoux** [comment]
Unless other people needing this, I won't do it. If someone wants to open a PR I'll merge it.

**Comment by Sharcoux** [comment]
crossingAllowed prop, fixed in version 6.1.0, is the best that we will provide. I'll close.
